src/Microsoft.Azure.WebJobs.Host/Listeners/ListenerFactoryContext.cs

```diff
@@ -39,6 +43,11 @@ public ListenerFactoryContext(FunctionDescriptor descriptor, ITriggeredFunctionE
         public ITriggeredFunctionExecutor Executor { get; private set; }
 
         /// <summary>
+        /// Gets the web jobs exception handler
+        /// </summary>
+        public IWebJobsExceptionHandler ExceptionHandler { get; private set; }
```

Comment:
meat of the change is adding the IWebJobsExceptionHandler to ListenerFactoryContext so that listeners can hook into the webjobs exception pipeline during normal execution (not just during `StartAsync`).